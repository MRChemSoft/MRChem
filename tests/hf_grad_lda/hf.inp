world_prec = 1.0e-3                 # Overall relative precision
world_size = 6                      # Size of simulation box 2^n

MPI {
  numerically_exact = true          # Guarantee identical results in MPI
}

Basis {
  order = 6                         # Polynomial order
}

Molecule {
$coords
H      0.0000  0.0000   0.0000
F      0.0000  0.0000   1.7300
$end
}

WaveFunction {
  method = DFT                      # Wave function method (HF or DFT)
}

DFT {
$functionals
LDA
$end
}

Properties {
  geometric_derivative = true       # Compute geometric derivative
}

SCF {
  kain = 3                        # Length of KAIN iterative history
  orbital_thrs = 1.0e-2           # Convergence threshold in orbital residual
  guess_type = SAD_DZ             # Type of initial guess: none, mw, gto
}
