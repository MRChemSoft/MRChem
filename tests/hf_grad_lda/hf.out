                                                                           
                                                                           
***************************************************************************
***                                                                     ***
***                                                                     ***
***          __  __ ____   ____ _                                       ***
***         |  \/  |  _ \ / ___| |__   ___ _ __ ___                     ***
***         | |\/| | |_) | |   | '_ \ / _ \ '_ ` _ \                    ***
***         | |  | |  _ <| |___| | | |  __/ | | | | |                   ***
***         |_|  |_|_| \_\\____|_| |_|\___|_| |_| |_|                   ***
***                                                                     ***
***         VERSION            1.1.0-alpha                              ***
***                                                                     ***
***         Git branch         mrchem-gradient                          ***
***         Git commit hash    6bcf101d1c739fbcb464-dirty               ***
***         Git commit author  Peter Wind                               ***
***         Git commit date    Fri Jul 13 10:37:14 2018 +0200           ***
***                                                                     ***
***         Contact: luca.frediani@uit.no                               ***
***                                                                     ***
***         Radovan Bast            Magnar Bjorgve                      ***
***         Roberto Di Remigio      Antoine Durdek                      ***
***         Luca Frediani           Gabriel Gerez                       ***
***         Stig Rune Jensen        Jonas Juselius                      ***
***         Rune Monstad            Peter Wind                          ***
***                                                                     ***
***************************************************************************

---------------------------------------------------------------------------

 MPI processes           :       (no bank)                               1
 OpenMP threads          :                                               4
 Total cores             :                                               4
                                                                           
---------------------------------------------------------------------------

XCFun DFT library Copyright 2009-2020 Ulf Ekstrom and contributors.
See http://dftlibs.org/xcfun/ for more information.

This is free software; see the source code for copying conditions.
There is ABSOLUTELY NO WARRANTY; not even for MERCHANTABILITY or
FITNESS FOR A PARTICULAR PURPOSE. For details see the documentation.
Scientific users of this library should cite
U. Ekstrom, L. Visscher, R. Bast, A. J. Thorvaldsen and K. Ruud;
J.Chem.Theor.Comp. 2010, DOI: 10.1021/ct100117s

---------------------------------------------------------------------------

 MRCPP version         : 1.2.0
 Git branch            : mrchem-gradient
 Git commit hash       : 6bcf101d1c739fbcb464-dirty
 Git commit author     : Peter Wind
 Git commit date       : Fri Jul 13 10:37:14 2018 +0200

 Linear algebra        : EIGEN v3.3.7
 Parallelization       : OpenMP

---------------------------------------------------------------------------


                                                                           
===========================================================================
                         MultiResolution Analysis
---------------------------------------------------------------------------
 polynomial order      : 6
 polynomial type       : Interpolating
---------------------------------------------------------------------------
 total boxes           : 8
 boxes                 : [          2           2           2 ]
 unit lengths          : [   32.00000    32.00000    32.00000 ]
 scaling factor        : [    1.00000     1.00000     1.00000 ]
 lower bounds          : [  -32.00000   -32.00000   -32.00000 ]
 upper bounds          : [   32.00000    32.00000    32.00000 ]
 total length          : [   64.00000    64.00000    64.00000 ]
===========================================================================


                                                                           
***************************************************************************
***                                                                     ***
***                        Initializing Molecule                        ***
***                                                                     ***
***************************************************************************
                                                                           
                                                                           
===========================================================================
                                 Molecule
---------------------------------------------------------------------------
 Charge                  :                                               0
 Multiplicity            :                                               1
---------------------------------------------------------------------------
    N    Atom            :               x               y               z
---------------------------------------------------------------------------
    0       H            :        0.000000        0.000000        0.000000
    1       F            :        0.000000        0.000000        1.730000
---------------------------------------------------------------------------
 Center of mass          :        0.000000        0.000000        1.642841
===========================================================================


                                                                           
***************************************************************************
***                                                                     ***
***                 Computing Ground State Wavefunction                 ***
***                                                                     ***
***************************************************************************
                                                                           
                                                                           
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 Calculation             : Compute initial orbitals
 Method                  : Diagonalize SAD Hamiltonian
 Precision               : 1.00000e-03
 Restricted              : True
 Functional              : LDA (SVWN5)
 AO basis                : Hydrogenic orbitals
 Zeta quality            : 2
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~


===========================================================================
                            Molecular Orbitals
---------------------------------------------------------------------------
 Alpha electrons         :                                               5
 Beta electrons          :                                               5
 Total electrons         :                                              10
---------------------------------------------------------------------------
    n  Occ Spin          :                                            Norm
---------------------------------------------------------------------------
    0    2    p          :                              9.999993801819e-01
    1    2    p          :                              9.999999966796e-01
    2    2    p          :                              9.999999695578e-01
    3    2    p          :                              9.999998993582e-01
    4    2    p          :                              9.999999158285e-01
===========================================================================


~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 Calculation             : Compute initial energy
 Method                  : DFT
 Precision               : 1.00000e-03
 Localization            : Off
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~


===========================================================================
                        Molecular Energy (initial)
---------------------------------------------------------------------------
 Kinetic energy          :            (au)                105.482034526492
 E-N energy              :            (au)               -257.033963952943
 Coulomb energy          :            (au)                 57.299099339316
 Exchange energy         :            (au)                  0.000000000000
 X-C energy              :            (au)                -10.452311670521
 Ext. field (el)         :            (au)                  0.000000000000
---------------------------------------------------------------------------
 N-N energy              :            (au)                  5.202312138728
 Ext. field (nuc)        :            (au)                  0.000000000000
---------------------------------------------------------------------------
 Electronic energy       :            (au)               -104.705141757656
 Nuclear energy          :            (au)                  5.202312138728
---------------------------------------------------------------------------
 Total energy            :            (au)             -9.950282961893e+01
                         :      (kcal/mol)             -6.243896777817e+04
                         :        (kJ/mol)             -2.612446413534e+05
                         :            (eV)             -2.707609810325e+03
===========================================================================


===========================================================================
                        Orbital Energies (initial)
---------------------------------------------------------------------------
    n  Occ Spin          :                                         Epsilon
---------------------------------------------------------------------------
    0    2    p          :            (au)                -23.605714370254
    1    2    p          :            (au)                 -0.974066782494
    2    2    p          :            (au)                 -0.363202469868
    3    2    p          :            (au)                 -0.212411706366
    4    2    p          :            (au)                 -0.212080107120
---------------------------------------------------------------------------
 Sum occupied            :            (au)                -50.734950872205
===========================================================================


~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 Calculation             : Optimize ground state orbitals
 Method                  : DFT
 Checkpointing           : Off
 Max iterations          : 100
 KAIN solver             : 3
 Localization            : Off
 Diagonalization         : First two iterations
 Start precision         : 1.00000e-03
 Final precision         : 1.00000e-03
 Helmholtz precision     : 1.00000e-03
 Energy threshold        : Off
 Orbital threshold       : 1.00000e-02
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~


===========================================================================
 Iter           MO residual             Total energy                Update
---------------------------------------------------------------------------
    0          2.236068e+00         -99.502829618928         -9.950283e+01
    1          2.640797e-01         -99.794946192935         -2.921166e-01
    2          1.430441e-01         -99.832048929301         -3.710274e-02
    3          8.636185e-02         -99.837472213390         -5.423284e-03
    4          3.704701e-02         -99.842524042313         -5.051829e-03
    5          1.217370e-02         -99.838440819715          4.083223e-03
---------------------------------------------------------------------------
                      SCF converged in 5 iterations!
===========================================================================


                                                                           
***************************************************************************
***                                                                     ***
***                    Printing Molecular Properties                    ***
***                                                                     ***
***************************************************************************
                                                                           
                                                                           
===========================================================================
                                 Molecule
---------------------------------------------------------------------------
 Charge                  :                                               0
 Multiplicity            :                                               1
---------------------------------------------------------------------------
    N    Atom            :               x               y               z
---------------------------------------------------------------------------
    0       H            :        0.000000        0.000000        0.000000
    1       F            :        0.000000        0.000000        1.730000
---------------------------------------------------------------------------
 Center of mass          :        0.000000        0.000000        1.642841
===========================================================================


===========================================================================
                         Molecular Energy (final)
---------------------------------------------------------------------------
 Kinetic energy          :            (au)                 99.426961508629
 E-N energy              :            (au)               -249.825405599547
 Coulomb energy          :            (au)                 55.501151861057
 Exchange energy         :            (au)                  0.000000000000
 X-C energy              :            (au)                -10.143460728582
 Ext. field (el)         :            (au)                  0.000000000000
---------------------------------------------------------------------------
 N-N energy              :            (au)                  5.202312138728
 Ext. field (nuc)        :            (au)                  0.000000000000
---------------------------------------------------------------------------
 Electronic energy       :            (au)               -105.040752958444
 Nuclear energy          :            (au)                  5.202312138728
---------------------------------------------------------------------------
 Total energy            :            (au)             -9.983844081972e+01
                         :      (kcal/mol)             -6.264956698457e+04
                         :        (kJ/mol)             -2.621257884336e+05
                         :            (eV)             -2.716742255937e+03
===========================================================================


===========================================================================
                         Orbital Energies (final)
---------------------------------------------------------------------------
    n  Occ Spin          :                                         Epsilon
---------------------------------------------------------------------------
    0    2    p          :            (au)                -24.078552545195
    1    2    p          :            (au)                 -1.093670009669
    2    2    p          :            (au)                 -0.495991277032
    3    2    p          :            (au)                 -0.357851248172
    4    2    p          :            (au)                 -0.357850476308
---------------------------------------------------------------------------
 Sum occupied            :            (au)                -52.767831112752
===========================================================================


===========================================================================
                           Dipole Moment (dip-1)
---------------------------------------------------------------------------
 r_O                     :        0.000000        0.000000        0.000000
---------------------------------------------------------------------------
 Electronic vector       :        0.000056        0.000246      -16.277802
 Magnitude               :            (au)                       16.277802
                         :         (Debye)                       41.374043
---------------------------------------------------------------------------
 Nuclear vector          :       -0.000000       -0.000000       15.570000
 Magnitude               :            (au)                       15.570000
                         :         (Debye)                       39.574989
---------------------------------------------------------------------------
 Total vector            :        0.000056        0.000246       -0.707802
 Magnitude               :            (au)                        0.707803
                         :         (Debye)                        1.799054
===========================================================================


===========================================================================
                           Geometric Derivative
---------------------------------------------------------------------------
 Total                   :        0.000078        0.000340       -2.980455
                         :        0.000114        0.000523       -0.157491
 Norm                    :            (au)                        2.984613
---------------------------------------------------------------------------
 Nuclear                 :        0.000000        0.000000        0.000000
                         :        0.000000        0.000000       -3.007117
 Norm                    :            (au)                        3.007117
---------------------------------------------------------------------------
 Electronic              :        0.000078        0.000340       -2.980455
                         :        0.000114        0.000523        2.849626
 Norm                    :            (au)                        4.123527
===========================================================================


                                                                           
                                                                           
***************************************************************************
***                                                                     ***
***                            Exiting MRChem                           ***
***                                                                     ***
***                       Wall time :  0h  1m 43s                       ***
***                                                                     ***
***************************************************************************
                                                                           
                                                                           
